Common/Data/Custom/Fred/CBOE.cs

```diff
@@ -0,0 +1,213 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+namespace QuantConnect.Data.Custom.Fred
+{
+    public static partial class FredSeries
+    {
+        public static class CBOE
+        {
+            ///<summary>
+            /// CBOE Equity VIX on Google (in Index)
+            /// Retrieved from FRED, Federal Reserve Bank of St. Louis; https://fred.stlouisfed.org/series/VXGOGCLS
+            /// </summary>
+            /// <remarks>
+            /// Copyright, 2016, Chicago Board Options Exchange, Inc. Reprinted with permission.
+            /// </remarks>
+            public static string VIXOnGoogle => "VXGOGCLS";
+
+            ///<summary>
+            /// CBOE DJIA Volatility Index (in Index)
+            /// Retrieved from FRED, Federal Reserve Bank of St. Louis; https://fred.stlouisfed.org/series/VXDCLS
+            /// </summary>
+            /// <remarks>
+            /// Copyright, 2016, Chicago Board Options Exchange, Inc. Reprinted with permission.
+            /// </remarks>
+            public static string VXD => "VXDCLS";
+
+            ///<summary>
+            /// CBOE Equity VIX on Goldman Sachs (in Index)
+            /// Retrieved from FRED, Federal Reserve Bank of St. Louis; https://fred.stlouisfed.org/series/VXGSCLS
+            /// </summary>
+            /// <remarks>
+            /// Copyright, 2016, Chicago Board Options Exchange, Inc. Reprinted with permission.
+            /// </remarks>
+            public static string VIXOnGoldmanSachs => "VXGSCLS";
+
+            ///<summary>
+            /// CBOE Equity VIX on IBM (in Index)
+            /// Retrieved from FRED, Federal Reserve Bank of St. Louis; https://fred.stlouisfed.org/series/VXIBMCLS
+            /// </summary>
+            /// <remarks>
+            /// Copyright, 2016, Chicago Board Options Exchange, Inc. Reprinted with permission.
+            /// </remarks>
+            public static string VIXOnIBM => "VXIBMCLS";
+
+            ///<summary>
+            /// CBOE Equity VIX on Amazon (in Index)
+            /// Retrieved from FRED, Federal Reserve Bank of St. Louis; https://fred.stlouisfed.org/series/VXAZNCLS
+            /// </summary>
+            /// <remarks>
+            /// Copyright, 2016, Chicago Board Options Exchange, Inc. Reprinted with permission.
+            /// </remarks>
+            public static string VIXOnAmazon => "VXAZNCLS";
+
+            ///<summary>
+            /// CBOE S&P 100 Volatility Index: VXO (in Index)
```

Comment:
Please check the comments for `&` and replace with the HTML entities version; 
https://stackoverflow.com/questions/19794929/vs2010-comments-with-an-ampersand-get-marked-as-warnings